### ISSUE TITLE
Poor accessibility of `Radio` due to missing `name` attribute

### ISSUE THREAD

**Issue by adamkudrna** [react-ui-org/react-ui#620]
Users cannot flip through individual `Radio` options with arrow keys.

It would be possible if all inputs in the same `Radio` group would have the same `name` attribute. However, current implementation doesn' allow that.
